README.md

```diff
@@ -120,4 +120,19 @@ cd Northwind365
 dotnet new ssw-ca
 ```
 
-> NOTE: This will use the Directory name as the Solution name and project namespaces.
+> NOTE: Not specifying `name` will use the Directory name as the Solution name and project namespaces.
```

Comment:
This is ok but I think the wording could be a little less confusing. Something like:

> Note: `name` is optional; if you don't specify it, the directory name will be used as the solution name and project namespaces.

Reply:
Done - I like what you've suggested, and have updated this.